olp-cpp-sdk-dataservice-read/src/repositories/CatalogRepository.cpp

```diff
@@ -184,6 +184,96 @@ CatalogVersionResponse CatalogRepository::GetLatestVersion(
   return version_response;
 }
 
+client::CancellationToken CatalogRepository::GetLatestVersion(
+    const CatalogVersionRequest& request, CatalogVersionCallback callback) {
+  repository::CatalogCacheRepository repository(
```

Comment:
```suggestion
  CatalogCacheRepository repository(
```

You are aldedy in repository namespace.